_posts/2021/2021-03-31-hmat_memkind.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Memkind support for heterogeneous memory attributes
+author: michalbiesek
+layout: post
+identifier: hmat_memkind
+---
+
+### Introduction
+
+Memkind is a library that provides support for Persistent Memory. However, it is not the only type of memory that memkind supports.
+It is a general solution designated for the platforms with heterogeneous memory.
+
+But before we go to heterogenous memory itself, let's start with a short recap about [NUMA][kernel-numa].
+The NUMA concept aims the problem of the dynamic extension of CPU count per socket and system memory.
```

Comment:
aims to solve?

tackles the problem of?

(I'm not sure what you are trying to say)